core/src/main/scala/kafka/api/FetchResponse.scala

```diff
@@ -138,7 +138,7 @@ class TopicDataSend(val dest: String, val topicData: TopicData) extends Send {
   buffer.rewind()
 
   private val sends = new MultiSend(dest,
-                            JavaConversions.seqAsJavaList(topicData.partitionData.toList.map(d => new PartitionDataSend(d._1, d._2))))
+    JavaConverters.seqAsJavaListConverter[Send](topicData.partitionData.map(d => new PartitionDataSend(d._1, d._2))).asJava)
```

Comment:
This is a bit annoying. Is that the most concise non-deprecated option available?